### ISSUE TITLE
run metrics in parallel

### ISSUE THREAD

**Issue by yegor256** [yegor256/jpeek#78]
Calculate metrics in parallel threads.

**Comment by yegor256** [comment]
@rultor release, tag is `0.19.1`

**Comment by 0crat** [comment]
Oops! There is no order for job `gh:yegor256/jpeek#78`.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/jpeek#78` is not in scope.
